src/App.js

```diff
@@ -0,0 +1,39 @@
+import './App.css';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
+import LoginPage from './pages/LoginPage';
+import RegisterPage from './pages/RegisterPage';
+import SplashPage from './pages/SplashPage';
+import NotFoundPage from './pages/NotFoundPage';
+import NewTherapistPage from './pages/NewTherapistPage';
+import HomePage from './pages/HomePage';
+import TherapistDetailsPage from './pages/TherapistDetailsPage';
+import ForgotPasswordPage from './pages/ForgotPasswordPage';
+import DeleteTherapist from './pages/DeleteTherapist';
+import NewAppointment from './pages/NewAppointment';
+import AppointmentPage from './pages/AppointmentPage';
+import Therapists from './components/Therapists';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor main `App` component in your project to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered a non-best practice.

- [ ] OPTIONAL: Refactor the other functions to ES6.  Tagging as optional because there are many.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.  

That way, your code will meet the latest standards. 💪